### PR TITLE
integration/docker: read default number of vCPUs from config file

### LINKED ISSUE #350: read runtime configuration file to get the default number of vCPUs
instead of run a container with `nproc` as workload to get the default number of vCPUs, we should read the configuration file since it is more reliable

### PR DESCRIPTION
**Pull request by devimc**
Read the default number of vCPUs from config file is more reliable
than runnning a container with nproc as workload

fixes #350

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  name = "github.com/BurntSushi/toml"
+  packages = ["."]
+  revision = "b26d9c308763d68093482582cea63d69be07a0f0"
```
Comment (grahamwhaley) [inline Gopkg.lock]: Any chance we could move this commit, maybe to the head commit. This is tied to the 0.3.0 release I believe, but I'd like to pick up [this commit](https://github.com/BurntSushi/toml/commit/4678bf20d0945128c23c31e748cde56476e8fd91) In particular to get the updated license for that package :-)

File: integration/docker/main_test.go [hunk 2]
```diff
@@ -17,11 +19,33 @@ const (
...
+
 func randomDockerName() string {
 	return RandID(30)
 }
 
 func TestIntegration(t *testing.T) {
+	var err error
+	runtimeConfigPath := DefaultRuntimeConfigPath
+
+	args := []string{"--kata-show-default-config-paths"}
```
Comment (jodh-intel) [inline integration/docker/main_test.go]: Nice

File: integration/docker/main_test.go [hunk 3]
```diff
@@ -17,11 +19,33 @@ const (
...
+	cmd := NewCommand(Runtime, args...)
+	stdout, _, exitCode := cmd.Run()
+	if exitCode == 0 && stdout != "" {
+		for _, c := range strings.Split(stdout, "\n") {
+			if _, err = os.Stat(c); os.IsExist(err) {
```
Comment (jodh-intel) [inline integration/docker/main_test.go]: This doesn't look right to me - I don't think `IsExist()` does what we want here as `err` is expected to be `nil` when `os.Stat()` succeeds.

File: Gopkg.lock (hunk 1 above)
Comment (devimc) [reply Gopkg.lock]: done, thanks

**Comment by devimc** [comment]
@jodh-intel @grahamwhaley  changes applied, thanks

**Comment by jodh-intel** [comment]
lgtm

**Comment by chavafg** [comment]
lgtm

**Pull request merged**